eos-tech-support/eos-enable-coredumps

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# Enables coredumps and adjusts storage path to work with coredumpcl
+
+if [ $# -lt 1 ] || [ $1 == "-h" ] || [ $1 == "--help" ]; then
+    echo "Usage:"
+    echo "   `basename $0` sysconfdir"
+    echo "Where:"
+    echo "   sysconfdir = path to the system configuration (e.g. /etc)"
+    exit
+fi
+
+SYSCONFDIR=${1}
```

Comment:
This could default to `/etc` if unspecified.

Reply:
Ok